.github/workflows/ci.yml

```diff
@@ -0,0 +1,11 @@
+name: CI
+
+on: push
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-20.04
```

Comment:
Why use an older Ubuntu here? I would go for `ubuntu-latest` or if explicit compatibility to older base OSes is required I'd add a matrix that tests with a few base OSes.

Reply:
sure, @reu-dev, what do you think ? You just chose it coz I mentioned it ? if so, I don't mind changing to latest. The latester the better 😆

Reply:
done